docs/src/pages/docs/monoids/First.md

```diff
@@ -65,6 +66,45 @@ chooseFirst([ 'a', 'b', 'c' ])
 
 </article>
 
+<article id="topic-construction">
+
+## Construction
+
+```haskell
+First :: a -> First (Maybe a)
+First :: Maybe a -> First (Maybe a)
+```
+
+There are two ways to construct a `First` instance using the constructor. The
+first requires that you provide any value (excluding a `Maybe a`) and you will
```

Comment:
first here should be `First`

Reply:
well it is the "first" of 2 ways of constructing.
Maybe the english should be a bit better.